it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/dataflow/DefaultPipelineLauncher.java

```diff
@@ -360,11 +360,22 @@ public LaunchInfo launch(String project, String region, LaunchConfig options) th
             options.executable() != null,
             "Cannot launch a dataflow job "
                 + "without executable specified. Please specify executable and try again!");
+        if (options.requirementsFile() != null) {
+          // install requirements
+          cmd.add(
+              "virtualenv . && source ./bin/activate && pip3 install -r "
+                  + options.requirementsFile());
+          cmd.add("&&");
+        }
```

Comment:
This will break on Windows -- I think this should be fine, but we should probably make a note somewhere on the supported/unsupported environment

Reply:
That's a good point. I'll add support for windows in a separate PR.